modules/@ergonode/ui/src/components/MenuList/MenuList.vue

```diff
@@ -42,6 +42,24 @@ export default {
             required: true,
         },
     },
+    computed: {
+        visibleMenu() {
+            return this.menu.filter(({
+                visible,
+            }) => {
+                if (typeof visible === 'string') {
+                    return this.$hasAccess([
```

Comment:
what about keeping unified structure as we have with routing? User is passing privileges which are allowing to visit page